tensorflow/lite/toco/graph_transformations/drop_fake_quant.cc

```diff
@@ -45,7 +45,7 @@ ::tensorflow::Status DropFakeQuant::Run(Model* model, std::size_t op_index,
   }
 
   // Drop min/max inputs
-  for (int i = 1, iter_limit = fakequant_op->inputs.size(); i < iter_limit;
+  for (int i = 1, end = fakequant_op->inputs.size(); i < end;
```

Comment:
No longer a need to split the loop header

Reply:
These are updated, the changes are reflected in the batch 11 PR.
https://github.com/tensorflow/tensorflow/pull/41167